templates/blocks/carousel_header_full-width-classic.twig

```diff
@@ -4,7 +4,7 @@
 {% endmacro %}
 
 <section class="block block-header block-wide carousel-header_full-width-classic">
-	<div id="carousel-wrapper-header"
+	<div id="old-carousel-wrapper-header"
```

Comment:
Didn't check everything, but I didn't see something in the new version that would collide with this. I guess we can preserve the `id`?

Reply:
The issue is that we use the id to add some things in the old version (indicators, arrow events, etc) so if it's the same id in the new version then there will be issues and the block won't behave as expected. Of course that would happen only if both versions of the block are on the same page which should never be the case, but I still think we should fix it just in case since it is technically possible. However maybe I can change the new block's id instead of the old one? 🤔

Reply:
Does the new version still use this id for something? I couldn't locate the same code (that used the id) as the previous version in the new version, or new code that uses it. But I might have missed it, this PR is massive :smile:

Reply:
That's a good point actually, the id is removed in the new version 😅 I guess something else fixed the issues I was seeing, I'll double-check!

Reply:
Removed the id change! 😄